src/htm/algorithms/SpatialPooler.hpp

```diff
@@ -148,20 +156,20 @@ class SpatialPooler : public Serializable
         columns the same regardless of the size of their receptive
         fields.
 	If numActivePerInhArea is specified then 
-	localAreaDensity must be < 0, and vice versa.
+	@ref localAreaDensity must be < 0, and vice versa.
 
-  @param stimulusThreshold This is a number specifying the minimum
+  @param stimulusThreshold This is a number specifying the minimum //TODO replace with `robustness` 0..1.0, which will affect this & synPermInc/Dec
         number of synapses that must be active in order for a column to
         turn ON. The purpose of this is to prevent noisy input from
         activating columns.
 
-  @param synPermInactiveDec The amount by which the permanence of an
+  @param synPermInactiveDec The amount by which the permanence of an //TODO make fixed and only depend on robustness?
         inactive synapse is decremented in each learning step.
 
-  @param synPermActiveInc The amount by which the permanence of an
+  @param synPermActiveInc The amount by which the permanence of an //TODO ditto
```

Comment:
TODO how related ActiveInc, InactiveDec? Something as "prefer forgetting, or learning new?" Which is about relative ratio of the two